codalab/rest/bundles.py

```diff
@@ -409,7 +410,7 @@ def _netcat_bundle(uuid, port):
     Send a raw bytestring into the specified port of the running bundle with uuid.
     Return the response from this bundle.
     """
-    check_bundles_have_all_permission(local.model, request.user, [uuid])
+    check_bundles_have_read_permission(local.model, request.user, [uuid])
```

Comment:
Change it back so that one can launch a service that others can hit (for demos).  In the future, we want to explicitly expose ports.

Reply:
Could you add a comment / TODO for this?

Reply:
Done